api/src/test/java/au/org/aodn/nrmn/restapi/service/SpreadSheetServiceIT.java

```diff
@@ -230,4 +230,13 @@ void duplicateRowsShouldBeRemoved() throws Exception {
         assertEquals(20, validRows.size());
     }
 
+    @Test
```

Comment:
Can you add comment to your test case so that what it is doing?

Reply:
> Can you add comment to your test case so that what it is doing?

Got it.